integration/run_test

```diff
@@ -26,6 +26,8 @@ workdir="/go/src/github.com/chef/automate"
 
 docker_run_args=(
         "--detach"
+        "--env" "CI"
```

Comment:
Hold up yo, does this mean that when we locally test it and a failure occurs that we'll auto-kill the container?

Reply:
it passes on the CI environment variable. so if you set it to true locally, then probably.